elements/pfe-accordion/src/pfe-accordion-panel.html

```diff
@@ -1,5 +1,5 @@
 <div tabindex="-1" role="tabpanel">
-  <div class="container">
+  <div class="container clearfix">
```

Comment:
I'm not a fan of helper classes in the context of web components because the code is all built inside the shadow DOM and in that way, the styles can't really be shared.  You can add the mixin to the container class directly.  What would you think of the following template updates:

```
<div class="pfe-accordion-panel__container" tabindex="-1" role="tabpanel">
  <slot></slot>
</div>
```

Honestly, I'm not convinced we even need the 2nd div here, does the layout break if we combine lines 1 and 2?

Reply:
Hmm, i don't know if we need both divs, but that would be out of scope for what this PR is trying to accomplish. I removed the clearfix class so now there are no edits to the template in this PR :)